src/components/app.tsx

```diff
@@ -114,6 +114,7 @@ export const authAndConnect = async (stores: IStores) => {
       stores.class.updateFromPortal(classInfo, includeAIUser);
     }
 
+    // RESEARCHER-ACCESS: should this be changed to isTeacherOrResearcher?
```

Comment:
I think no - we're not going to create `user` documents in the database for researchers.

Reply:
I mean... unless it's straightforward. It would be useful.